cns/cniconflist/generator.go

```diff
@@ -63,6 +66,11 @@ type CiliumGenerator struct {
 	Writer io.WriteCloser
 }
 
+// SWIFTGenerator generates the Azure CNI conflist for the SWIFT scenario
+type SWIFTGenerator struct {
```

Comment:
Just curious, how/when we use this generation of conflist ?

Reply:
it will be used in aks swift